lib/vty.h

```diff
@@ -182,6 +182,11 @@ struct vty_arg
 /* Small macro to determine newline is newline only or linefeed needed. */
 #define VTYNL  ((vty->type == VTY_TERM) ? "\r\n" : "\n")
 
+/* for compatibility */
+#define VTY_NEWLINE                               VTYNL
```

Comment:
for compatibility with other code that is integrated with FRR (and Quagga) and to minimize the cost of moving from Quagga to FRR